src/services/Fetch/Fetch.ts

```diff
@@ -0,0 +1,55 @@
+import axios from "axios";
+import {
+  LOCAL_FILES_PATHS,
+  REMOTE_DATA_SOURCE_URL,
+} from "../../../utils/constants.js";
+import * as fs from "fs";
+import * as path from "path";
+
+export class Fetch {
+    public fileName: string;
+    protected remoteAdress: string;
+    protected rawData: any;
+
+  /**
+   * Create a fetcher.
+   * @param {string} fileName - Store base name of remote file.
+   * @const {string} remoteAdress - remote https adress stored in constant.ts file
+   * @const {any} rawData - row data downloaded from remote website
+   */
+
+  constructor() {
+    this.fileName = null;
+    this.remoteAdress = REMOTE_DATA_SOURCE_URL;
+    this.rawData = null;
+  }
+
+  error(message: string): never {
+    throw new Error(message);
+  }
+
+  /** 
+   * Test remote address and save fetched data
+   * @return {void} 
+   */
+
+  async download() {
+    try {
+      const response: any = await axios({
+        url: REMOTE_DATA_SOURCE_URL,
+        method: "get",
+        responseType: "stream",
+      });
+      if (response.status == "200") {
```

Comment:
Petit saut de ligne :
```suggestion
      });
      
      if (response.status == "200") {
```

Reply:
dac